src/main/scala/cromwell/binding/RuntimeAttributes.scala

```diff
@@ -79,6 +82,10 @@ object RuntimeAttributes {
     value map validateBoolean getOrElse Defaults.FailOnStderr.successNel
   }
 
+  private def validateFailOnRc(value: Option[String]): ValidationNel[String, Boolean] = {
+    value map validateBoolean getOrElse Defaults.FailOnRc.successNel
```

Comment:
The `getOrElse` is handling the case where nothing was passed in, so it's just the `SuccessNel` of our default value